Data Structures/Arrays/2D Array - DS/array ds/arrayds.java

```diff
@@ -0,0 +1,58 @@
+mport java.io.*;
```

Comment:
Correct this line. It should be `import` not `mport`.

Reply:
okay got it